src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,25 @@
 package core.basesyntax;
 
+import core.basesyntax.shapes.Drawable;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
 
+    private static final int SHAPES_LIMIT = 10;
+
+    public static void main(String[] args) {
+        // minimum size of array = 1
+        Drawable[] shapeArray = new Drawable[(int) (Math.random() * SHAPES_LIMIT) + 1];
```

Comment:
```suggestion
        Shape[] shapeArray = new Shape[(int) (Math.random() * SHAPES_LIMIT) + 1];
```
you should be able to call `draw()` on a Shape

Reply:
FIXED